integration-tests/ui-integration-tests/src/test-setup-scripts/utils/child-process-manager.js

```diff
@@ -53,5 +53,10 @@ module.exports = {
             stdio: 'ignore',
             detached: true
         }).unref();
+        
+        // spawn('node', ['src/test-setup-scripts/stop-selenium.js'], {
```

Comment:
I commented them as they will be needed when we have UI automation and they are still under development